client/dashboard/sites/settings-site-visibility/index.tsx

```diff
@@ -1,15 +1,15 @@
 import { useQuery, useMutation } from '@tanstack/react-query';
+import { useParams } from '@tanstack/react-router';
 import { Card, CardBody } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { siteQuery, siteSettingsMutation, siteSettingsQuery } from '../../app/queries';
-import { siteSettingsSiteVisibilityRoute } from '../../app/router';
 import PageLayout from '../../components/page-layout';
 import SettingsPageHeader from '../settings-page-header';
 import { LaunchForm } from './launch-form';
 import { PrivacyForm } from './privacy-form';
 
 export default function SiteVisibilitySettings() {
-	const { siteSlug } = siteSettingsSiteVisibilityRoute.useParams();
+	const { siteSlug } = useParams( { strict: false } );
```

Comment:
`strict: false` is needed otherwise `useParams` throws an error saying that the param `from` is needed. Adding the `from` defeats the purpose of this PR since it's the route path.